accord-core/src/test/java/accord/impl/list/ListStore.java

```diff
@@ -49,28 +147,162 @@ public ListStore(Node.Id node)
         this.node = node;
     }
 
-    public Timestamped<int[]> get(Key key)
+    public synchronized Timestamped<int[]> get(Ranges unavailable, Timestamp executeAt, Key key)
```

Comment:
`synchronized` was from a different patch and only kept it here as `write` had it...